Apromore-Core-Components/Apromore-Portal/src/main/java/org/apromore/portal/ConfigBean.java

```diff
@@ -122,7 +126,12 @@ public ConfigBean(String siteEditor, String siteExternalHost, int siteExternalPo
         this.enableSubscription = enableSubscription;
 
         this.enableCalendar     = enableCalendar;
-        this.maxUploadSize      = maxUploadSize;
+
+        this.maxUploadSize = maxUploadSize;
+
+        LoggerFactory.getLogger(getClass()).info("\n\n>>> >>> > useKeycloakSso {}:", useKeycloakSso);
```

Comment:
Either merge this with the previous log message, or remove it as debugging code.

Reply:
Merged with previous log message - this is now redundant message (i.e. removed) [DONE].